Makefile

```diff
@@ -258,3 +258,8 @@ dev/up: deploy/manifests
 .PHONY: dev/down
 dev/down:
 	source ./scripts/local-dev.sh && down
+
+.PHONY: test-e2e $(driver)
```

Comment:
I'm not sure about this Makefile syntax, have you double-checked this?
Especially with the `.PHONY` directive, I wouldn't expect this.

I'm assuming this would work like `make test-e2e driver=kvm2`, Would it work without specifying any args?

Reply:
Yep, if no argument is passed, like `make test-e2e`, it just falls back to the default driver(virtualbox) specified in the `local-e2e.sh` script

Reply:
Since this hasn't used anywhere in the PR, I'm trusting your word on this ;)